src/game-container/MainContainer.svelte

```diff
@@ -4,7 +4,7 @@
     import UIContainer from './game-ui-container/UIContainer.svelte'
     import TxtContainer from './game-txt-container/TxtContainer.svelte'
 
-	export let gameState;
+    export let gameState;
```

Comment:
Je export 2x een let gameState, is dat gewenst? kan de rest vd file niet zien of misschien doe ik iets fout maar kan me vorstellen dat je hier ene gamestate update doet. Zou je ook bv een aparte functie voor kunnen maken, een soort updateGameState(newGameState) wat dan in een aparte file staat waar je de gamestate weer uit exporteerd, dan heb je geen risico's. Die gamestate zou dan soort prop in die file zijn die je erbuiten niet mag gebruiken en alleen dmv update, get, etc achtige methodes. Beetje database idee